<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Builder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/appicon</normaloff>:/icons/appicon</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="mainStack">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="mainPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Line</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Path</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsPage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="progressPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Script"/>
   <addaction name="actionRe_scan_script"/>
   <addaction name="actionBuild"/>
  </widget>
  <widget class="QStatusBar" name="mainStatusBar"/>
  <action name="actionOpen_Script">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open</normaloff>:/icons/open</iconset>
   </property>
   <property name="text">
    <string>Open script...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCollect_Files">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Collect files...</string>
   </property>
   <property name="toolTip">
    <string>Collect all needed files in one folder and update paths in the script.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/export</normaloff>:/icons/export</iconset>
   </property>
   <property name="text">
    <string>Build...</string>
   </property>
   <property name="toolTip">
    <string>Build the script in a single file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionStringify_Binary">
   <property name="text">
    <string>Stringify binary...</string>
   </property>
   <property name="toolTip">
    <string>Stringify binary</string>
   </property>
  </action>
  <action name="actionRe_scan_script">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/scan</normaloff>:/icons/scan</iconset>
   </property>
   <property name="text">
    <string>Re-scan script</string>
   </property>
   <property name="shortcut">
    <string>F5, Ctrl+R</string>
   </property>
  </action>
  <action name="actionIgnoreJSDoc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ignore /** */</string>
   </property>
  </action>
  <action name="actionIgnoreBlockComments">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ignore /* */</string>
   </property>
  </action>
  <action name="actionIgnoreLineComments">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ignore //</string>
   </property>
  </action>
  <action name="actionKeepLicense">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Keep /* License */</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/cogs</normaloff>:/icons/cogs</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionBuild_JSDoc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Build JSDoc</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/help_m</normaloff>:/icons/help_m</iconset>
   </property>
   <property name="text">
    <string>Get help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionBug_Report">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/bugreport</normaloff>:/icons/bugreport</iconset>
   </property>
   <property name="text">
    <string>Bug Report</string>
   </property>
   <property name="toolTip">
    <string>Report a bug / Request a feature</string>
   </property>
  </action>
  <action name="actionChat">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/chat</normaloff>:/icons/chat</iconset>
   </property>
   <property name="text">
    <string>Chat</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
